build-system/server/new-server/transforms/transform.ts

```diff
@@ -23,12 +23,14 @@ import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
 const FOR_TESTING = argv._.includes('integration');
+const HOST = argv.host ? argv.host : '0.0.0.0';
 // Use 9876 if running integration tests as this is the KARMA_SERVER_PORT
-const PORT = FOR_TESTING ? 9876 : 8000;
+const PORT = FOR_TESTING ? 9876 : (argv.port ? argv.port : 8000);
 const ESM = !!argv.esm;
 
 const defaultTransformConfig = {
   esm: ESM,
+  host: HOST,
```

Comment:
Host is not static, it's per request. Eg, hosting on `0.0.0.0` (the now default) allows connections on `192.168.1.100` (or whatever your local IP is). So test on a phone would request `http://192.168.1.100:8000`, and we should respond with URLs using that requesting domain.

Reply:
i went ahead and remove the overriding host code and removed reading from global argv